paddle/fluid/operators/generator/tests_utils.py

```diff
@@ -89,16 +89,14 @@ def is_tensor_list(s):
 def exist_mutable_attribute(attrs):
     for attr in attrs:
         if (
-            attr['typename'] in ['Scalar', 'IntArray']
-            and attr['support_tensor'] is True
-        ):
+            is_scalar(attr['typename']) or is_intarray(attr['typename'])
+        ) and attr['support_tensor'] is True:
```

Comment:
attr['support_tensor'] is True -> attr['support_tensor']

Reply:
done